doc/sphinx/user_manual/basic_starter_tutorial/02_your_first_input_file.md

```diff
@@ -1,23 +1,41 @@
-(part:user_manual:chap:concepts:sec:02_first_input_file)=
+(part:user_manual:chap:basic_starter_tutorial:sec:02_first_input_file)=
 Your first input file
 =====================
 
+GWB input files are plain-text files, which generally have the extension `.wb`. At minimum, the world builder input file should contain a object (indicated by `{}` in JSON) containing a version number (JSON key: `version`) and a list (indicated by `[]` in JSON) of features (JSON key: `features`). This results in the following input file:  
 
-```{todo}
-Explaining the minimum wb input file (and what it produces) and include a picture of the final result for both composition and temperature.
-```
 
 
-```{literalinclude} ../../_static/gwb_input_files/BST_1_minimal_box.wb
+```{literalinclude} ../../_static/gwb_input_files/BST_02_minimal_box.wb
 :language: json
 :lineno-start: 1
 ```
 
-::::{grid} 2
-:::{grid-item-card}  Download BST_1_minimal_box.wb
-:link: ../../_static/gwb_input_files/BST_1_minimal_box.wb
+::::{grid} 3
+:::{grid-item-card} BST_1_minimal_box.wb
+:link: ../../_static/gwb_input_files/BST_02_minimal_box.wb
+:::
+:::{grid-item-card} BST_1_minimal_box.grid
+:link: ../../_static/gwb_input_files/BST_02_minimal_box.grid
 :::
-:::{grid-item-card}  Download BST_1_minimal_box.grid
-:link: ../../_static/gwb_input_files/BST_1_minimal_box.grid
+:::{grid-item-card} Pavaview v1 state file 
+:link: ../../_static/paraview_state_files/BST_v1.pvsm
 :::
 ::::
+
+```{note}
+You can copy the text in the box by selecting it, or by hovering over the box and clicking the copy symbol in the top right corner. The file can also be downloaded by clicking on the `Download BST_1_minimal_box.wb` button below the textbox. 
+
+If you want to inspect the result yourself, you can also download the corresponding `.grid` file and use the `gwb-grid` application (`gwb-grid BST_1_minimal_box.wb BST_1_minimal_box.grid`) to create a vtk output (`BST_1_minimal_box.vtk`) and view it in for example Paraview.
+```
+
+Congratulations on creating your first world builder input file! If you visualize the result you will notice that you are getting an adiabatic temperature profile and that every compositional field is zero everywhere. This is the background, or our canvas, if we want to stay in the painting analogy. Now, before we are going to get our brushes, we should first discuss the shape of our canvas a bit more.
+
+
+```{figure} ../../../../doc/sphinx/_static/images/user_manual/basic_starter_tutorial/BST_02.png
```

Comment:
The top part of the figure it is not supposed to show anything because all compositions are zero. In the figure capture I described it as `Currently all compositional fields are zero everywhere, so nothing is shown`. I am not sure how I can write this more clearly, suggestions are welcome :)

Reply:
I realized this once I got a lot further into the tutorial, forgot I had written this here. I did provide an alternative wording in on my of comments/suggestions somewhere below. I think the confusion comes from using "zero" vs composition "0". Let me know if you can't find my suggestion or it doesn't sound right to you

Reply:
Found it in my lower comment: Maybe re-write this as "The top part of the figure shows where the composition as been assigned as an object." or something like that that avoids the "zero" vs "0" confusion. <- Might be useful for all the figures. But in this first figure, maybe explicitly write "The top part of the figure shows where the composition has been assigned as object. Since no compositions have been assigned yet, nothing is plotted when we view the composition solution. The bottom part shows the temperature solution that has been assigned". Just so it is clear to the viewer that the figure is being displayed correctly?

Reply:
sounds good, I will make the change.